docs/data_structures.rst

```diff
@@ -44,3 +39,29 @@ Lattice
 
 .. autoclass:: mbuild.Lattice
     :members:
+
+
+Load
+--------------------
+
+.. automodule:: mbuild.conversion.load
+    :members:
+
+
+Load CIF
+--------------------
+
+.. automodule:: mbuild.lattice.load_cif
+    :members:
+
```

Comment:
```suggestion
```
These are functions, not data structures, so I wouldn't include them here.

Reply:
This was moved to a new load data tab